Cargo.toml

```diff
@@ -12,6 +12,13 @@ license = "MIT OR Apache-2.0"
 [build-dependencies]
 build_const = "0.2"
 
+#[dev-dependencies]
```

Comment:
shall we remove unrelated changes or you want to move benchmarks to use criterion instead in this pr?

Reply:
Criteron is nice, but we can use whatever you want.  I left both in for you to choose

Reply:
cool, let's remove the commented code and revisit benchmark framework in separate pr?

Reply:
Removed, Criterion adds a lot of bloat.  I'll save the code for later in case it needs added back in